install.sh

```diff
@@ -74,8 +74,6 @@ make_virtualenv() {
     fi
 
     echo ""
-    echo "===== Checking dependencies for conflict ..."
-    python3 -m pip check && echo "No conflicts" || exit 1
```

Comment:
same as above. Would be better to put it into a separated PR and keep the commit history cleaner, but we can squeeze it into this one as well I think.

Reply:
should I revert this change

Reply:
nah, just a note